packages/angular/src/jsonforms-root.component.ts

```diff
@@ -70,9 +71,9 @@ export class JsonForms implements OnChanges, OnInit {
         });
         this.jsonformsService.$state.subscribe(state => {
             const data = state?.jsonforms?.core?.data;
-            if (data !== this.data) {
-                this.dataChange.emit(data);
-            }
+            const errors = state?.jsonforms?.core?.errors;
+            this.dataChange.emit(data);
+            this.errors.emit(errors);
```

Comment:
Hmm doesn't this emit the data and errors for every state changed, even when they specifically did not change? For example when the validationMode or locale is set or a renderer is added/removed?